hamcrest-matchers/java8-hamcrest-matchers/src/main/java/it/ozimov/cirneco/hamcrest/java8/J8Matchers.java

```diff
@@ -38,4 +40,39 @@
         return Matchers.not(IsEmptyOptional.emptyOptional());
     }
 
+    /**
+     * Creates a matcher from an inner matcher for {@linkplain Optional}s which are present.
+     */
+    public static <T> TypeSafeMatcher<Optional<T>> presentAnd(final Matcher<? super T> innerMatcher) {
```

Comment:
J8Matchers is an aggregator for all the matchers. The inner static class should be a standalone class.
Do you want to do the change? Oherwise I can do it